build/run.js

```diff
@@ -198,7 +198,7 @@ commands.build.rust = async function (argv) {
         console.log('Minimizing the WASM binary.')
         await gzip(paths.wasm.main, paths.wasm.mainGz)
 
-        const limitMb = 4.65
+        const limitMb = 4.03
```

Comment:
How did it happen? I'm concerned right now. Is this number different on your system than other systems?

Reply:
As discussed with @farmaazon, I'm lowering the limit as a result of deleting several unused embedded fonts.